catalog/ACA/ACA_reef_habitat_v2_0.jsonnet

```diff
@@ -0,0 +1,322 @@
+    on v1.0 and new technical developments in mapping methodology. A brief overview of 
+    the changes can be found 
+    [here](https://allencoralatlas.org/blog/geomorphic-and-benthic-maps-2022-update/) 
+    and a more comprehensive technical summary can be found 
+    [here](https://storage.googleapis.com/coral-atlas-static-files/resources-page-materials/Allen_Coral_Atlas_2022_habitat_map_revisions.pdf).
+
+    The Allen Coral Atlas was funded by [Vulcan Inc.](https://vulcan.com/) and is 
+    managed by the [Arizona State University Center for Global Discovery and 
+    Conservation Science](https://gdcs.asu.edu/). Partners include 
+    [Planet](https://www.planet.com/), the 
+    [University of Queensland](https://www.uq.edu.au/), and the 
+    [Coral Reef Alliance](https://coral.org/en/).
+    
+    Scientific background publications:
+
+    - Lyons, M. B., Roelfsema, C. M., Kennedy, E. V., Kovacs, E. M., Borrego-Acevedo, R., Markey, K., ... & Murray, N. J. (2020). Mapping the world's coral reefs using a global multiscale earth observation framework. Remote Sensing in Ecology and Conservation, 6(4), 557-568. [doi:10.1002/rse2.157](https://doi.org/10.1002/rse2.157)
+
+    - Kennedy, E. V., Roelfsema, C. M., Lyons, M. B., Kovacs, E. M., Borrego-Acevedo, R., Roe, M., ... & Tudman, P. (2021). Reef Cover, a coral reef classification for global habitat mapping from remote sensing. Scientific Data, 8(1), 1-20. [doi:10.1038/s41597-021-00958-z](https://doi.org/10.1038/s41597-021-00958-z)
+
+    - Roelfsema, C. M., Lyons, M., Murray, N., Kovacs, E. M., Kennedy, E., Markey, K., ... & Phinn, S. R. (2021). Workflow for the generation of expert-derived training and validation data: a view to global scale habitat mapping. Frontiers in Marine Science.  [doi:10.3389/fmars.2021.643381](https://doi.org/10.3389/fmars.2021.643381)
+
+    - Li, J., Knapp, D. E., Lyons, M., Roelfsema, C., Phinn, S., Schill, S. R., & Asner, G. P. (2021). Automated global shallow water bathymetry mapping using Google Earth Engine. Remote Sensing, 13(8), 1469. [doi:10.3390/rs13081469](https://doi.org/10.3390/rs13081469)
+
+    - Li, J., Knapp, D. E., Fabina, N. S., Kennedy, E. V., Larsen, K., Lyons, M. B., ... & Asner, G. P. (2020). A global coral reef probability map generated using convolutional neural networks. Coral Reefs, 39(6), 1805-1815. [doi:10.1007/s00338-020-02005-6](https://doi.org/10.1007/s00338-020-02005-6)
+
+    Allen Coral Atlas maps, bathymetry and map statistics are &copy; 2023 Allen
+    Coral Atlas Partnership and Vulcan, Inc.
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id) + [
+    ee.link.license(license.reference),
+    {
+      rel: ee_const.rel.source,
+      href: 'https://storage.googleapis.com/coral-atlas-user-downloads-prod/Global-Dataset-20211006223100.zip',
+    },
+    {
+      rel: ee_const.rel.cite_as,
+      href: 'https://doi.org/10.5281/zenodo.3833242',
+    },
+  ],
+  keywords: [
```

Comment:
Please sort the keywords

Issue('ACA/reef_habitat/v2_0', 'ACA/ACA_reef_habitat_v2_0.json', 'keywords', '"keywords" must be sorted', IssueLevel.error)